Remotion/Web/Development.WebTesting/WebDriver/Configuration/Edge/ExtendedEdgeOptions.cs

```diff
@@ -16,19 +16,22 @@
 // 
 using System;
 using JetBrains.Annotations;
+using Microsoft.Edge.SeleniumTools;
 using OpenQA.Selenium.Chrome;
 
 namespace Remotion.Web.Development.WebTesting.WebDriver.Configuration.Edge
 {
   /// <summary>
   /// Extends <see cref="ChromeOptions"/> provided by Selenium with additional data used by the web testing infrastructure.

```

Comment:
If you adapt this comment and the one after this you can remove `using OpenQA.Selenium.Chrome;` too